config/settings.yml

```diff
@@ -17,6 +17,8 @@ current_recruitment_cycle_year: 2020
 govuk_notify:
   api_key: please_change_me
   welcome_email_template_id: please_change_me
+  course_update_email_template_id: please_change_me
+  course_base_url: please_change_me
```

Comment:
Can you stick local host for find please

It more like find_url as it's the base for generating course url. 

Then we can move it to its own and not under govuk_notify

Reply:
Is it find or publish url? It's looking at the course page, but from publish rather than find.

Reply:
opps pub thats what i meant